common/src/main/scala/models/Notification.scala

```diff
@@ -0,0 +1,14 @@
+package models
```

Comment:
Notification-related models shouldn't be in `common` project, they are not relevant to `registration` module